midend/expandLookahead.cpp

```diff
@@ -78,7 +79,7 @@ DoExpandLookahead::ExpansionInfo *DoExpandLookahead::convertLookahead(
               em->method);
     auto targ = expression->typeArguments->at(0);
     auto typearg = typeMap->getTypeType(targ, true);
-    if (!typearg->is<IR::Type_StructLike>() && !typearg->is<IR::Type_Tuple>()) return nullptr;
+    if (!SymbolicValueFactory(typeMap).isFixedWidth(typearg)) return nullptr;
```

Comment:
Tests are failing because this is changing many test outputs. Will move this to a separate PR.

Reply:
Moved to #4264. Will continue the testing of this PR once that PR is merged.

Reply:
It looks really weird to depend on SymbolicValueFactory in this pass.
Why can't the isFixedWidth function be a method of the typeMap, for example?

Reply:
Makes sense. I'll refactor it to be a method of typeMap.

Reply:
Done in #4264.